specs/deliverly_server_SUITE.erl

```diff
@@ -8,20 +8,27 @@
 
 init_per_suite(Config) ->
   lager:start(),
-  ulitos_app:set_var(?APP, default_app, true),
+  ulitos_app:set_var(?APP, redis_database, 7),
   Config.
 
 end_per_suite(_) ->
   application:stop(lager),
   ok.
 
-init_per_group(_, Config) ->
+init_per_group(Group, Config) ->
+  Auth =
+    case Group of
+      clients_auth_referer -> {de_auth_referer, ["example.io", "example.org"]};
+      _ -> {de_auth_token, []}
+    end,
+  ulitos_app:set_var(?APP, default, #{auth => [Auth]}),
   deliverly:start(),
   test_app_app:start([],[]),
   timer:sleep(200),
   Config.
 
-end_per_group(_,Config) ->
+end_per_group(_, _Config) ->
+  redis_cli:q(["EVAL", "return redis.call('del', unpack(redis.call('keys', ARGV[1])))",  0,  "*"]),
```

Comment:
Line 31 is too long:   redis_cli:q(["EVAL", "return redis.call('del', unpack(redis.call('keys', ARGV[1])))",  0,  "*"]),.